platform/accessibility/508-accessibility-best-practices.md

```diff
@@ -31,9 +34,14 @@
 1. Axe Scans
 
    1. Front-end engineering should install the [axe plugin for Chrome or Firefox](https://deque.com/axe) and run it periodically during their daily work.
-   2. This manual process should be repeated in end-to-end tests that scan rendered pages for regressions every time a build is initiated. These e2e 508 scans should be looking for `['Section 508', 'WCAG 2 Level A', 'WCAG 2 Level AA']` errors.
+   2. This manual process should be repeated in automated processes that scan rendered pages for regressions every time a build is initiated. These e2e 508 scans should be looking for `['Section 508', 'WCAG 2 Level A', 'WCAG 2 Level AA']` errors.
```

Comment:
You might spell out end-to-end here instead of below then be able to just use the acronym after this point.